packages/backend-modules/auth/migrations/sqls/20241220130020-removeTwitterFacebook-up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE users
+DROP COLUMN "twitterHandle";
+DROP COLUMN "facebookId";
+DROP COLUMN "publicUrl";
```

Comment:
same here, `DROP` statements need to be separated by commas

Reply:
done